tests/unit/cli/inspect/test_metadataprinter.py

```diff
@@ -0,0 +1,142 @@
+import pandas as pd
+from pandas.testing import assert_frame_equal
+
+from csvcubed.cli.inspect.metadataprinter import MetadataPrinter
+from csvcubed.models.csvwtype import CSVWType
+from csvcubed.models.cube.cube_shape import CubeShape
+from csvcubed.models.inspectdataframeresults import (
+    DatasetObservationsByMeasureUnitInfoResult,
+    DatasetObservationsInfoResult,
+)
+from csvcubed.models.sparqlresults import CodelistsResult
+from csvcubed.utils.iterables import first
+from tests.helpers.inspectors_cache import get_data_cube_inspector
+from tests.unit.cli.inspect.test_inspectdatasetmanager import (
+    expected_dataframe_pivoted_multi_measure,
+    expected_dataframe_pivoted_single_measure,
+)
+from tests.unit.test_baseunit import get_test_cases_dir
+
+_test_case_base_dir = get_test_cases_dir() / "cli" / "inspect"
+
+
+def test_multi_measure_pivoted_shape_observation_info():
```

Comment:
We're also testing the value count information as well. Could you include that in the name/description or else split this unit test into one which tests the obs info and the other which tests the value counts?

Could you also make clear in the description that this is testing a cube (as opposed to a code list).